packages/types/src/xlucene-interfaces.ts

```diff
@@ -0,0 +1,21 @@
+export enum XluceneFieldType {
```

Comment:
It is xLucene not Xlucene

```suggestion
export enum xLuceneFieldType {
```